dask_bigquery/tests/test_core.py

```diff
@@ -82,3 +82,16 @@ def test_read_kwargs(dataset, client):
 
     with pytest.raises(Exception, match="504 Deadline Exceeded"):
         ddf.compute()
+
+
+def test_read_columns(df, dataset, client):
```

Comment:
Overall this test looks great. One small suggestions: could we add

```python
assert df.shape[1] > 1
```

to ensure that the original `DataFrame` has more than one column? Otherwise, in the future the example `DataFrame` could be updated to only have a single `"name"` column and this test would still pass (this is unlikely, but possible)